src/store/audienceCache.ts

```diff
@@ -0,0 +1,58 @@
+import { StorageKeys, AudienceDefinition, CachedAudienceMetaData } from 'types';
+import { storage, timeStampInSecs } from '../utils';
+
+class AudienceCache {
+  cachedAudiences: AudienceDefinition[];
+  cachedAudiencesMetaData: CachedAudienceMetaData;
+
+  constructor() {
+    this.cachedAudiences = [];
+    this.cachedAudiencesMetaData = {
+      timeChecked: 0,
```

Comment:
timeChecked, sounds like this is related to matching / checking - why not `storedAt` or `cachedAt` this follows some of the other naming patterns.